RELEASE-NOTES.md

```diff
@@ -1,5 +1,44 @@
 # Prefect Release Notes
 
+## Release 2.6.5
+
+### Enhancements
+- Add support for manual flow run retries — https://github.com/PrefectHQ/prefect/pull/7152
```

Comment:
This needs QA and may be omitted from this release

Reply:
QA was done and it looks good.